LoadingUI/LoadingUI/LoadingViewController.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  LoadingViewController.swift
+//  LoadingUI
+//
+//  Created by Rob Vance on 10/5/20.
+//
+
+import UIKit
+
+class LoadingViewController: UIViewController {
```

Comment:
It might be nice to have a little control over the appearance of this LoadingViewController. Also, some documenation on how to use it would be helpful.